play/src/pusher/services/AdminApi.ts

```diff
@@ -1022,6 +1022,34 @@ class AdminApi implements AdminInterface {
         });
         return response.data ? response.data : [];
     }
+
+    /**
+     * @openapi
+     * /members:
```

Comment:
that is correct ?
If we search by `memberUUID`, we could use "/members/{uuid}"?

Reply:
I added the missing parameter for openApi documentation.